enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -83,7 +88,7 @@ const (
 
 	// Workspace slack space is how much extra space will be allocated in the
 	// workspace disk image beyond the size of the existing files.
-	workspaceSlackBytes = 1e9
+	workspaceSlackBytes = 100 * 1e6 // 100MB
```

Comment:
is this effectively a limit on the size of the outputs or am i misunderstanding what this is?

Reply:
Until fuse lands, yup :(